docs/openapi/api-eservice-operations-interop-v1.yaml

```
@@ -365,6 +365,61 @@ paths:
                 detail: 'There is a problem processing the request on the server.'
                 traceId: 'string'
                 errors: [{'code':'000-1234', 'detail':'The server is currently down for maintenance. Please try again later.'}]
+  '/eservices/pollingEserviceActive':
+    get:
+      summary: Retrive services with active state
+      description: >- 
+        Retrive services with active state
+      tags:
+        - Read
+      operationId: getEservicesActive
```

This is not simply returning active eservices, but applies specific polling filters 
```suggestion
      operationId: getEservicesReadyForPolling
```